src/test/visit_test_main.py

```diff
@@ -397,12 +397,15 @@ def GenFileNames(test_case, ext):
 #
 #  Programmer: Cyrus Harrison
 #  Date:       Wed May 30 2012
+#
+#  Mark C. Miller, Fri Sep 11 18:58:34 PDT 2020
+#  Add _dpix, _davg args
 # ----------------------------------------------------------------------------
-def CalcDiffState(p_pixs, d_pixs, davg):
+def CalcDiffState(p_pixs, d_pixs, davg, _dpix, _davg):
```

Comment:
I think these names (_dpix and _davg) aren't very clear, especially since they aren't documented in the docstring and there are very similarly named incoming vars. It's not a particularly big deal in this function because of it's small size, but it might be worth considering a change here.

Reply:
Ok, fixed this and the other case you mentioned.